backend/de.metas.material/planning/src/main/java/de/metas/material/planning/pporder/impl/PPRoutingRepository.java

```diff
@@ -311,7 +322,7 @@ public void changeRouting(@NonNull final PPRoutingChangeRequest changeRequest)
 
 			for (final I_AD_WF_Node routingActivityRecord : routingActivityRecords)
 			{
-				final PPRoutingActivityId activityId = PPRoutingActivityId.ofAD_WF_Node_ID(routingId, routingActivityRecord.getAD_WF_Node_ID());
+				final PPRoutingActivityId activityId = PPRoutingActivityId.ofAD_WF_Node_ID(routingId, routingRecord.getAD_WF_Node_ID());
```

Comment:
why this change?
i think `routingActivityRecord` shall be used

Reply:
By mistake. undo-redo then wrong copy paste. And then I didn't notice it. Thanks for find it!